Add the elasticity flexure module.

alarshi opened the pull request:
This PR is same as the version uploaded and reviewed on HubZero. Currently, the visualization notebook is not working on HubZero because of installation problems of the meshio library.

alarshi committed:
Add the elasticity flexure module.

alarshi committed:
Correct a reference.

naliboff commented on bin/elasticity-flexture-L0/0_overview-elasticity-flexure.ipynb:
```diff
@@ -0,0 +1,124 @@
+{
```
<p>"An introduction to the viscoselastic" -&gt; remove "the"</p><p> </p><p>Add "a" to "Bending of plates under <strong>a</strong> line load"</p><p> </p><p>Instead of "would be", maybe say "will be" or "should be"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geodynamics-hubzero/gem-notebooks/pull/15/discussion/#comment-2114815540'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='1'/>

naliboff commented on bin/elasticity-flexture-L0/0_overview-elasticity-flexure.ipynb:
```diff
@@ -0,0 +1,124 @@
+{
```
<p>Does the Dannberg, J. 2020 reference have a doi? I'm just wondering what the standard procedure is for citing lecture notes that have not been published, but provided as material that can be reused (I'm not sure).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geodynamics-hubzero/gem-notebooks/pull/15/discussion/#comment-2114815542'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

alarshi replied:
<p>No, I couldn't find any DOI associated with it. Yeah, I am not sure either. </p>

naliboff commented on bin/elasticity-flexture-L0/1_visco_elastic_deformation.ipynb:
```diff
@@ -0,0 +1,112 @@
+{
```
<blockquote>At shallow depths, where both temperatures and pressures are low, rocks dominantly deform elastically. </blockquote><p>Is this statement always correct? If stresses are high, rocks may be deform in a brittle manner at low temperatures and pressures.</p><blockquote>The existence of lithospheric structures that are over one billion years old suggests that the Earth's lithosphere behaves elastically on geological time scales.</blockquote><p>I'm not sure I follow the logic here. At long time scales, the Earth behaves in a viscous fashion, but some structures representing older brittle deformation are preserved.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geodynamics-hubzero/gem-notebooks/pull/15/discussion/#comment-2114815548'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

alarshi replied:
<p>&gt;Is this statement always correct? If stresses are high, rocks may be deform in a brittle manner at low temperatures and pressures.</p><p>Good point. I will add this as well.</p><p> </p><p>&gt;I'm not sure I follow the logic here. At long time scales, the Earth behaves in a viscous fashion, but some structures representing older brittle deformation are preserved.</p><p>You are right. I have removed the sentence. It was a bit confusing anyway.</p>

naliboff commented on bin/elasticity-flexture-L0/1_visco_elastic_deformation.ipynb:
```diff
@@ -0,0 +1,112 @@
+{
```
<p>Suggested rephrasing of last sentence: "....&nbsp;how long it takes for stresses in a deformed viscoelastic system to decay to a steady state"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geodynamics-hubzero/gem-notebooks/pull/15/discussion/#comment-2114815549'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='3'/>

naliboff commented on bin/elasticity-flexture-L0/2_bending_of_lithosphere_under_line_loads.ipynb:
```diff
@@ -0,0 +1,405 @@
+{
```
<p>Is there a source credit for this image?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geodynamics-hubzero/gem-notebooks/pull/15/discussion/#comment-2114815564'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

alarshi replied:
I made it but based it on lecture notes I found online. Not sure how to cite the source :(

naliboff commented on bin/elasticity-flexture-L0/2_bending_of_lithosphere_under_line_loads.ipynb:
```diff
@@ -0,0 +1,405 @@
+{
```
<p>The image in this cell is not showing up (at least in ReviewNB)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geodynamics-hubzero/gem-notebooks/pull/15/discussion/#comment-2114815567'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='2'/>

alarshi replied:
<p>Hmm, there is a typo again. It turns out that I messed up the files between the push here and the version on Hubzero :(</p><p> </p>

naliboff commented on bin/elasticity-flexture-L0/2_bending_of_lithosphere_under_line_loads.ipynb:
```diff
@@ -0,0 +1,405 @@
+{
```
<p>The image here is also not showing up in the review.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geodynamics-hubzero/gem-notebooks/pull/15/discussion/#comment-2114815568'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='12'/>

alarshi replied:
Thanks for pointing it out! Yes, there was a typo in the image name.

naliboff commented on bin/elasticity-flexture-L0/2_bending_of_lithosphere_under_line_loads.ipynb:
```diff
@@ -0,0 +1,405 @@
+{
```
<p>Typo: "deatils"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geodynamics-hubzero/gem-notebooks/pull/15/discussion/#comment-2114815569'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='16'/>

naliboff commented on bin/elasticity-flexture-L0/3_modeling_plate_bending.ipynb:
```diff
@@ -0,0 +1,254 @@
+{
```
<p>Capitalize "t" in the sentence starting with "<strong>t</strong>hese loads"</p><p> </p><p>Use "Kyr", "Ka", or "thousand of year" instead of "K" after "90"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geodynamics-hubzero/gem-notebooks/pull/15/discussion/#comment-2114815591'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='3'/>

naliboff reviewed (changes requested):
@alarshi - Thanks for this and overall it is looking great! I made a number initial of suggestions and edits via ReviewNB.

One general question - should the APSECT model parts be L1, or did we decide that would still be L0?

alarshi committed:
Address comments from the PR.

alarshi commented:
@naliboff : Thanks for the review and sorry for the delay in addressing your suggestions! I have addressed most suggestions except on how to cite Dannberg's model developed for the course that is used here. Let me know if you have ideas.

naliboff reviewed (approved):
Thanks for addressing the comments and I think this is ready to merge.

>  I have addressed most suggestions except on how to cite Dannberg's model developed for the course that is used here. Let me know if you have ideas.

Let's discuss with Juliane at some point in the near future, I don't think we need to come up with answer before getting this merged.
